### ISSUE TITLE
[RFE] [rancher-logging] Prefix systemDefaultRegistry to all images

### ISSUE THREAD

**Issue by mouellet** [rancher/rancher#43421]
### Is your feature request related to a problem? Please describe.

Right now most of the images used have repositories and tag values found in the chart's value.yaml and those values are automatically prefixed w/ the `{{ template "system_default_registry" . }}`.

Some of them, found in the logging crd, are missing. Here's what I've found, there might be others.. :
- `fluentbit.bufferVolumeImage`
- `fluentd.bufferVolumeImage`
- `fluentd.volumeModImage`
- `fluentd.scaling.drain.image`
- `fluentd.scaling.drain.pauseImage`

### Describe the solution you'd like
For all images used in the chart to be automatically prefixed w/ the system default registry.

### Describe alternatives you've considered
The values can be specified explicitly, but would required chart consumer to dig for the new/updated images tag or repository on upgrades.

Ex.:
```
  loggingOverlay:
    spec:
      fluentbit:
        bufferVolumeImage:
          repository: my.private-registry.io/banzaicloud/custom-runner
          tag: 0.1.0
      fluentd:
        bufferVolumeImage:
          repository: my.private-registry.io/banzaicloud/custom-runner
          tag: 0.1.0
        volumeModImage:
          repository: my.private-registry.io/busybox
          tag: latest
```

**Comment by jbiers** [comment]
@pankajsqe FYI I've just merged the v2.9 PR, just waiting for reviews on the v2.8 backport now

**Comment by pankajsqe** [comment]
Pass : prefix systemDefaultRegistry is appearing on all the logging images.

Details
Rancher : v2.9.1-rc3
Cluster : v1.30.1+rke2r1


![Image](https://github.com/user-attachments/assets/91d77017-6529-4b61-873b-2442ea42789b)
![Image](https://github.com/user-attachments/assets/56316e50-c9ed-43ff-801a-d3f67496b401)
![Image](https://github.com/user-attachments/assets/18ff2cfe-7bab-42d3-a36b-3884d224fa52)
![Image](https://github.com/user-attachments/assets/c28f8ab9-a469-4ae6-aaa2-9d703898a5b3)
